examples/Workflows/dog_breed_classification/README.md

```diff
@@ -0,0 +1,70 @@
+# Workflow pipeline example for classifying dog breed
+
+This workflow uses two models, first to classify if an image is of a dog or a cat. If it is a dog the second model does breed classification.
+To change the default batch size and batch delay the yaml file for the workflow can to be changed. This cannot currently be set via the REST API.
+
+## Flow
+
```

Comment:
@maaquib As discussed offline, please check if input can go to a preprocessing node in the workflow DAG and then be sent to cat-dog classification, this has happened in the [dual-translation](https://github.com/maaquib/serve/tree/workflow_nlp_example/examples/Workflows/nmt_tranformers_pipeline#example-2-dual-translation) in NLP example. In case this would be possible, it would unlock the other workflows such as parallel models with aggregated/multiple outputs

Reply:
Done